FactoryMethod/Projects/Logistics/src/Logistics.cpp

```diff
@@ -0,0 +1,15 @@
+//
+// Created by polia on 17.02.2019.
+//
+
+#include "Logistics.h"
+
+Logistics::Logistics() = default;
+
+Logistics::~Logistics() = default;
+
+void Logistics::planDelivery(const Cargo& cargo, const Destination& dest) const {
+	Transport* transport = createTransport();
```

Comment:
@PolinaShlepakova, is it good that you have to create and delete truck for each delivery?
Could you use some truck storage?
Or, maybe `truck pool`... :)

Reply:
No, it's not good. I thought about using some king of storage, hope you'll help me :)

Reply:
I will ;)